Configurations/10-main.conf

```diff
@@ -1355,12 +1355,14 @@ my %targets = (
         perlasm_scheme   => "auto",
         multilib         => "-x64",
     },
-    "VC-WIN32" => {
+    "VC-WIN64-ARM" => {
+        inherit_from     => [ "VC-WIN64-common" ],
+        cflags           => add("-D_ARM64_WINAPI_PARTITION_DESKTOP_SDK_AVAILABLE"),
+    },
+    "VC-WIN32-common" => {
         # x86 Win32 target defaults to ANSI API, if you want UNICODE,
         # configure with 'perl Configure VC-WIN32 -DUNICODE -D_UNICODE'
-        inherit_from     => [ "VC-noCE-common", asm("x86_asm"),
-                              sub { $disabled{shared} ? () : "uplink_common" } ],
-        cflags           => add("-WX"),
+        inherit_from     => [ "VC-noCE-common" ],
         as               => sub { vc_win32_info()->{as} },
         asflags          => sub { vc_win32_info()->{asflags} },
         asoutflag        => sub { vc_win32_info()->{asoutflag} },
```

Comment:
Just as in #4918, these should move to `VC-WIN32`, along with `perlasm_scheme`

Reply:
No is it appropriate to drag unicows thing along... [Actually I even wonder if unicows would actually work...] And speaking of which, there is no point in having non-UNICODE build for anything other that x86. Well, one can even argue that there is no point having non-UNICODE anywhere, since we are not actually supporting Win9x...

Reply:
> [Actually I even wonder if unicows would actually work...]

I meant "would actually work *now*".